package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
What I'm personally trying to avoid is having to fork and/or create PRs for every case like this in the future.  If this is unavoidable due to the way ES modules work, then I guess from now on, for every transpiled ES6 module, I'll just have to add the `{format: "es6", main: "src/index"}' overrides?  Although, I'm curious as to why I've never run into this problem with `webpack`, no matter what combination of `cjs` or `es6` modules, Babel plugins, etc.